From da5b9c2fb0338e616f5376b6e40b4531fa3d7e1c Mon Sep 17 00:00:00 2001
From: Matthias Hopf <mhopf@suse.de>
Date: Mon, 3 Aug 2009 17:17:44 +0200
Subject: [PATCH] Fix autoconf issues.

Apparently, autoconf 2.64 has "interesting" issues with spaces in double
quoted strings, as used in configure.ac. Change that to parantheses.
Based on a patch by Jens Lody <radeonhd@jenslody.de>.

Additionally, autogen.sh didn't --force so far.
---
 autogen.sh   |    4 ++--
 configure.ac |    6 +++---
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index d288ab4..58cf072 100644
--- a/configure.ac
+++ b/configure.ac
@@ -51,19 +51,19 @@ AH_BOTTOM([#endif])
 
 AC_ARG_WITH(xorg-module-dir,
             AC_HELP_STRING([--with-xorg-module-dir=DIR],
-                           [Default xorg module directory [[default=$libdir/xorg/modules]]]),
+                           [Default xorg module directory (default $libdir/xorg/modules)]),
             [moduledir="$withval"],
             [moduledir="$libdir/xorg/modules"])
 
 AC_ARG_ENABLE(exa,
               AC_HELP_STRING([--disable-exa],
-	                     [Disable EXA support [[default enabled]]]),
+	                     [Disable EXA support (default enabled)]),
               [EXA="$enableval"],
               [EXA=yes])
 
 AC_ARG_ENABLE(dri,
               AC_HELP_STRING([--disable-dri],
-	                     [Disable DRI support [[default enabled]]]),
+	                     [Disable DRI support (default enabled)]),
               [DRI="$enableval"],
               [DRI=yes])
 
-- 
1.6.3.3

